up42/initialization.py

```diff
@@ -36,7 +36,11 @@
 
 def _check_deprecated_implicit_project_id(project_id: Optional[str]):
     if not project_id:
-        warn("Provide the project ID as the value of the `project_id` argument.", DeprecationWarning, stacklevel=2)
```

Comment:
This does not come from my PR, where's the PR which changed this?

Reply:
Your PR was merged. What changed is that the pre-commit hook on my end breaks down single lines into multiple lines. It is a recurrent change on my end automatically when I run the pre-commit hook.

Reply:
This is what you changed in your PR. same thing but single line. Not sure if single line - multi line issue is too relevant, but I think multiline makes it more readable, however no strong preference on my side.

![Screenshot 2023-11-02 at 08 56 25](https://github.com/up42/up42-py/assets/3291933/044f1e63-03d7-4ffb-a2ad-da4927ca5337)

Reply:
This comes from a PR made to the parent branch by @dustgalactic. Take into account that she is involved in all the logs and communication to the users. I revert the change but JFYI. :)